pocs/utils/logger.py

```diff
@@ -45,25 +46,34 @@ def get_root_logger(profile='panoptes', log_config=None):
     # Get log info from config
     log_config = log_config if log_config else load_config('log').get('logger', {})
 
+    invoked_script = os.path.basename(sys.argv[0])
     log_dir = '{}/logs'.format(os.getenv('PANDIR', '/var/panoptes/'))
-    log_fname = 'panoptes-{}-{}.log'.format(os.getpid(), datetime.datetime.utcnow().strftime('%Y%m%dT%H%M%SZ'))
-    log_fname_generic = 'panoptes.log'
+    log_fname = '{}-{}-{}'.format(invoked_script, os.getpid(),
+                                  datetime.datetime.utcnow().strftime('%Y%m%dT%H%M%SZ'))
```

Comment:
Let me just say (whine) how much I hate timezones.

Reply:
We try to do everything UTC but I was tempted to change this to local time. All the timestamps in file are in UTC but you might want to know when the log file was generated locally. Probably not a big deal though.